app/views/green_lanes/category_assessments/_search_form.html.erb

```diff
@@ -8,7 +8,7 @@
                              label: { text: 'Commodity Code'} %>
 
       <%= f.govuk_collection_select :country,
-                                    TradingPartner.options,
+                                    GreenLanes::CategoryAssessmentSearch.country_options,
                                     :id,
                                     :name,
```

Comment:
this should change to `:long_description` which avoids then needing to map all the data into an openstruct

Reply:
Done